### DISCUSSION TITLE
Some field corners are missed

### DISCUSSION THREAD

**Discussion by Samus1111111** [Courseplay/Courseplay_FS22#1611] [Q&A] [answered]
Hello, I'm pretty new to courseplay with FS22. I've watched a number of videos and read about getting started, but I've noticed that on fields with some ~45 degree corners, I can't force CP to do a sharp turn and the smooth turn misses the actual corner. You can see this in the screenshots below from Haut-Beyleron field 43.

https://steamcommunity.com/sharedfiles/filedetails/?id=2812936030
https://steamcommunity.com/sharedfiles/filedetails/?id=2812936043

I have the "Headland Corners" set to sharp, but this happens with every tool. The path generation is generating a smooth corner which doesn't hit the actual corner.

**Comment by Tensuko** [comment, marked as answer]
That is becouse of the corner angle.
I suggest you to use field margin and extend the field over the border a bit.

As this is a "normal" problem.
![round corners](https://user-images.githubusercontent.com/27344744/170531334-f8e9a186-05ed-49c1-9bc9-870241050a90.png)
![round pattern](https://user-images.githubusercontent.com/27344744/170531340-a8b02ffa-5e58-4f66-b50d-e30e12707954.png)
